model/device.go

```diff
@@ -93,6 +94,138 @@ const (
 	DevTypeTest       = "Test"
 )
 
+// Consts containing defaults for all device configs.
+const (
+	defaultActPeriod int64 = 60
+	defaultMonPeriod int64 = 60
+)
+
+// Consts containing defaults for a controller config.
+const (
+	defaultControllerInputs  string = "A0,X50,X51,X60,X10"
+	defaultControllerOutputs string = "D14,D15,D16" // Peripheral Power.
+)
+
+// controllerDefaultVars is a map which relates variable names to their values for
```

Comment:
Perhaps "controllerDefaultVars is a map which relates controller variable names to their default values" would be more concise.